<ui version="4.0" >
 <comment>
*********************************************************************************
**
**   $Id$
**   Copyright (c) 2007 Joe Croft joe@croftj.net
**   
**   This file is part of QcjData Class Libraries.
**
**   QcjData Class Libraries is free software; you can redistribute it and/or modify
**   it under the terms of the GNU General Public License as published by
**   the Free Software Foundation; either version 2 of the License, or
**   (at your option) any later version.
**
**   QcjData Class Libraries is distributed in the hope that it will be useful,
**   but WITHOUT ANY WARRANTY; without even the implied warranty of
**   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
**   GNU General Public License for more details.
**
**   You should have received a copy of the GNU General Public License
**   along with QcjData Class Libraries; if not, write to the Free Software
**   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
**
*********************************************************************************
</comment>
 <class>QcjPhotoSelect</class>
 <widget class="QDialog" name="QcjPhotoSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Photo Select</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QSplitter" name="splitter" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>3</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeView" name="treeView" />
        <widget class="QGraphicsView" name="graphicsView" />
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adjustBtn" >
       <property name="text" >
        <string>Adjust Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>QcjPhotoSelect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>478</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel" >
     <x>424</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>QcjPhotoSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>537</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel" >
     <x>553</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
